types/ember/test/ember-module-tests.ts

```diff
@@ -0,0 +1,280 @@
+import Ember from 'ember';
+import EngineInstance from '@ember/engine/instance';
+import Component from '@ember/component';
+import EmberObject from '@ember/object';
+import ComputedProperty from '@ember/object/computed';
+import Evented from '@ember/object/evented';
+// $
+Ember.$; // $ExpectType JQueryStatic
+// A
+Ember.A(); // $ExpectType NativeArray<{}>
+Ember.A([1, 2]); // $ExpectType NativeArray<number>
+// addListener
+Ember.addListener({ a: 'foo' }, 'a', {}, () => {});
+Ember.addListener({ a: 'foo' }, 'a', null, () => {});
```

Comment:
add test with `once` set

Reply:
add test with method = string

Reply:
add test with target = Function

Reply:
These are all beyond the scope of what this change is attempting to test